report_helpers/__manifest__.py

```diff
@@ -2,12 +2,11 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    'name': 'Report Helpers',
-    'summary': "Helper methods and functional fields for easier "
-               "report generation",
-    'version': '12.0.0.0.0',
-    'license': 'AGPL-3',
-    'author': 'Open For Small Business Ltd',
-    'website': 'https://o4sb.com',
-    'depends': ['base', 'account'],
+    "name": "Report Helpers",
+    "summary": "Helper methods and functional fields for easier " "report generation",
+    "version": "12.0.0.0.0",
```

Comment:
bump version to 12.0.1.00
```suggestion
    "version": "12.0.1.0.0",
```